middle_end/flambda/inlining_transforms.ml

```diff
@@ -674,5 +673,6 @@ let inline_by_copying_function_declaration
       in
       let expr = Flambda_utils.bind ~body ~bindings:state.let_bindings in
       let env = E.activate_freshening (E.set_never_inline env) in
+      let env = E.set_inline_debuginfo ~dbg env in
```

Comment:
This shouldn't be added here. This will add a spurious stack frame to specialised function calls

Reply:
It is now removed.